app/views/shared/_navbar.html.erb

```diff
@@ -26,6 +26,9 @@
             </div>
           </li>
         <% else %>
+          <li class="nav-item">
+            <%= link_to "Offers", offers_path, class: "nav-link" %>
+          </li>
```

Comment:
Do we need this again in the navbar? We already have the Get Started button on the home page which we can use to launch into the offers page

Reply:
let me pull the latest version and check what's going on. the navbar i had at the time only featured login.